tests/jaeger_report.rs

```diff
@@ -0,0 +1,28 @@
+use holochain_tracing::*;
+
+extern crate crossbeam_channel;
+extern crate rustracing;
+extern crate rustracing_jaeger;
+
+use std::{thread, time::Duration};
+
+use holochain_tracing::tracer_jaeger::new_jaeger_tracer;
+pub use rustracing::sampler::*;
+pub use rustracing_jaeger::{Result, Span as RtSpan, *};
+
+#[test]
+fn report_test() {
+    let tracer = new_jaeger_tracer("report_test_33");
+    let parent_span: HSpan = tracer.span("parent").start().into();
+    {
+        for i in 0..30 {
+            thread::sleep(Duration::from_millis(100));
```

Comment:
This is a cool test but it will be expensive to run this in practice. Is there a quicker version or should this test be marked as an integration test?

Reply:
Its not for auto-testing. Maybe I should make this as a bin instead...